test/specs/test1.js

```diff
@@ -0,0 +1,33 @@
+import { expect, browser, $ } from '@wdio/globals';
+
+describe('Test 1 - Perform Login', () => {
+  it('Should log in to the "Swag Labs" page', async () => {
+    await browser.url(`https://www.saucedemo.com/`);
```

Comment:
lets store `https://www.saucedemo.com/` in baseUrl

Reply:
Done. Thanks.